FAQ.md

```diff
@@ -95,6 +94,18 @@ source ~/miniconda3/etc/profile.d/conda.sh
 
 and then try rerunning and/or reinstalling the Toolkit. 
 
+## How are partial charges assigned in a SMIRNOFF force field?
+
+There are [many charge methods](https://openforcefield.github.io/standards/standards/smirnoff/#partial-charge-and-electrostatics-models) supported by the SMIRNOFF specification. With the exception of water, mainline OpenFF force fields only use AM1-BCC (through `ToolkitAM1BCC`) to assign partial charges. (A future biopolymer force field will likely use library charges for standard residues.)
+
+If OpenEye Toolkits are installed and licensed, the ELF10 variant of AM1-BCC is used. OpenEye's OMEGA is used to generate conformers and OpenEye's Quacpac (`oequacpac.OEAM1BCCELF10Charges`) is used to generate partial charges.
+
+Otherwise, RDKit is used to generate a conformer which is passed to AmberTool's `sqm` (with `-c bcc`).
+
+Note that, because of differences with the ELF10 variant and other subtle differences between OpenEye Toolkits and RDKit/AmberTools, **assigned partial charges can be expected to differ** based on the available toolkit(s). These numerical differences are often minor but in some molecules or use cases can be significant.
+
+A future charge method may use [NAGL](https://github.com/openforcefield/openff-nagl) to assign partial charges from a graph-convolutional neural network instead of an underlying semi-empirical method.. As of March 2024, this is under development and not release for general use.
```

Comment:
```suggestion
A future charge method may use [NAGL](https://github.com/openforcefield/openff-nagl) to assign partial charges from a graph-convolutional neural network instead of an underlying semi-empirical method. This approach is anticipated to be faster, more scalable, and more consistent than current approaches. As of March 2024, this is under development and not released for general use.
```